### DISCUSSION TITLE
What plotting program do you use?

### DISCUSSION THREAD

**Discussion by vpinon** [Xyce/Xyce#37] [Q&A]
Hello,

Once you have your simulation results, do you use an interactive waveform viewer?
When trying to understand a circuit operation and performance, you pick new waves, zoom to some areas, add markers...
I'm used to Mentor EZwave, Cadence ADE (AWD/VIVA); I have tried GnuPlot, GWave and Labplot, but find none efficient for everyday tasks...
Do you have suggestions?

Thanks!

**Comment by tvrusso** [comment]
This is probably a good question to pose on the Xyce Google group.  I think the set of people who actually read Github discussions is pretty small.  There is a more diverse set on the Google group and you'll probably get more response there.

I don't have any good suggestions for you, as I pretty much only use Gnuplot.  

Xyce's "probe" or "csd" output formats should be usable in some Cadence viewers (which is why they exist).
